maths/minkowski_distance.py

```diff
@@ -0,0 +1,41 @@
+def minkowski_distance(
+    point_a: list[float],
+    point_b: list[float],
+    order: int,
+) -> float:
+    """
+    This function calculates the Minkowski distance for a given order between
+    two n-dimensional points represented as lists. For the case of order = 1,
+    the Minkowski distance degenerates to the Manhattan distance. For
+    order = 2, the usual Euclidean distance is obtained.
+
+    https://en.wikipedia.org/wiki/Minkowski_distance
+
+    >>> minkowski_distance([1.0, 1.0], [2.0, 2.0], 1)
+    2.0
+    >>> minkowski_distance([1.0, 2.0, 3.0, 4.0], [5.0, 6.0, 7.0, 8.0], 2)
+    8.0
+    >>> minkowski_distance([1.0], [2.0], -1)
+    Traceback (most recent call last):
+        ...
+    Exception: The order must be greater than or equal to 1.
+    >>> minkowski_distance([1.0], [1.0, 2.0], 1)
+    Traceback (most recent call last):
+        ...
+    Exception: Both points must have the same dimension.
+    """
+    if order < 1:
+        raise Exception("The order must be greater than or equal to 1.")
```

Comment:
```suggestion
        raise ValueError("The order must be greater than or equal to 1.")
```
Please make the error of a more specific type